### PR TITLE
V5

### PR DESCRIPTION
**Pull request by AlbertoElias**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,190 +1,97 @@
...
-```
+* data-o-header-search: Applied to the search `<form>`. Inside, it also requires an `<input>` element
+* data-o-header-togglable: Applied to any toggle in the header
+* data-o-header-togglable-nav: Applied to the `<button>` or other kind of element that will toggle the meganav
+* data-o-header-togglable-search: Applied to the `<button>` or other kind of element that will toggle the search input
+* data-o-header-suggestions: Applied to the root div of the bottom row of the header, as that's the container of the suggestions that are added via JS
+* data-o-header-edition-switch-button: Applied to the `<button>` or other kind of element that will toggle the edition switch
+* data-o-header-selectable: Applied to the `<button>` or other kind of element in the meganav section in the mobile view that will select it
 
-## Upgrading from 3.x.x to 4.x.x
```
Comment (onishiweb) [inline README.md]: Is it worth adding an `Upgrading from 4.x.x to 5.x.x` section?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+	git clean -fxd
+
+install:
+	npm install
+	obt install --verbose
+
+verify:
+	obt verify
+
+test: verify
```
Comment (onishiweb) [inline Makefile]: Should there also be `obt test` as there are tests to be run?

File: README.md (hunk 1 above)
Comment (AlbertoElias) [inline README.md]: it is a complete overhaul. Do you think it's worth having a section to mention that? Happy to add it

File: Makefile (hunk 2 above)
Comment (AlbertoElias) [inline Makefile]: The tests are in the circle.yml. I've got no idea how this Makefile got here, I'll remove it

File: README.md [hunk 3]
```diff
@@ -1,190 +1,97 @@
-# Header [![Build Status](https://travis-ci.org/Financial-Times/o-header.svg)](https://travis-ci.org/Financial-Times/o-header)
```
Comment (onishiweb) [inline README.md]: Would it be worth adding information about the different themes available as well?

Comment (AlbertoElias) [inline README.md]: Yup! Cheers

File: README.md (hunk 1 above)
Comment (onishiweb) [inline README.md]: I've seen people asking about that fairly often when we do a new version. So maybe it's worth just having an `Upgrading to 5.x.x` even if it's just "throw everything out and start over".

File: README.md (hunk 3 above)
Comment (j-mes) [inline README.md]: Are you not using the status API token from Circle CI for the shield or is that unneeded?

File: Makefile (hunk 2 above)
Comment (onishiweb) [inline Makefile]: ha, ok then. Glad I mentioned it.

File: README.md (hunk 3 above)
Comment (AlbertoElias) [inline README.md]: @j-mes that's a removed line, the readme is using the circle badge

**Comment by AlbertoElias** [comment]
@onishiweb better?

**Comment by onishiweb** [comment]
Yep, looks great! :shipit:

**Pull request merged**